yourgov/components/Staff/StaffMembersTable.tsx

```diff
@@ -37,9 +75,46 @@ export const StaffMembersTable = ({
 		sort,
 	});
 
+	const [successMessageType, setSuccessMessageType] =
+		useState<null | successType>(router.query.successType as successType);
+	const [updatedStaffMemberName, setUpdatedStaffMemberName] = useState(() => {
+		const updatedStaffMember = staffMembersGetState().find(
+			(staffMember) => router.query.staffId === staffMember?.id
+		);
+
+		return updatedStaffMember?.name || 'the staff member';
+	});
+
 	return (
 		<SortAndFilterProvider value={sortAndFilter}>
-			<DataProvider value={data}>
+			<DataProvider
+				value={{
+					...data,
+					setSuccessMessageType,
+					setUpdatedStaffMemberName,
+				}}
+			>
+				{successMessageType && (
+					<SectionAlert
+						focusOnUpdate={[successMessageType, updatedStaffMemberName].join(
```

Comment:
This doesn't seem to work. No focus was given for Pause or Remove

Reply:
Looks like I fixed batch actions but broke individual actions. I'll fix this.

- [ ] Focusing alert on update not working for individual actions, only batch actions